tests.py

```diff
@@ -697,54 +697,113 @@ def test_get_semicoherent_nglitch_twoF(self):
 
 class MCMCSearch(Test):
     label = "TestMCMCSearch"
+    h0 = 1
+    sqrtSX = 1
+    F0 = 30
+    F1 = -1e-10
+    F2 = 0
+    minStartTime = 700000000
+    duration = 1 * 86400
+    maxStartTime = minStartTime + duration
+    Alpha = 5e-3
+    Delta = 1.2
+    tref = minStartTime
+
+    def test_fully_coherent_MCMC_fixedsky(self):
 
-    def test_fully_coherent(self):
-        h0 = 1
-        sqrtSX = 1
-        F0 = 30
-        F1 = -1e-10
-        F2 = 0
-        minStartTime = 700000000
-        duration = 1 * 86400
-        maxStartTime = minStartTime + duration
-        Alpha = 5e-3
-        Delta = 1.2
-        tref = minStartTime
         Writer = pyfstat.Writer(
-            F0=F0,
-            F1=F1,
-            F2=F2,
+            F0=self.F0,
+            F1=self.F1,
+            F2=self.F2,
             label=self.label,
-            h0=h0,
-            sqrtSX=sqrtSX,
+            h0=self.h0,
+            sqrtSX=self.sqrtSX,
             outdir=self.outdir,
-            tstart=minStartTime,
-            Alpha=Alpha,
-            Delta=Delta,
-            tref=tref,
-            duration=duration,
+            tstart=self.minStartTime,
+            Alpha=self.Alpha,
+            Delta=self.Delta,
+            tref=self.tref,
+            duration=self.duration,
             Band=4,
         )
-
         Writer.make_data()
+
         predicted_FS = Writer.predict_fstat()
 
         theta = {
-            "F0": {"type": "norm", "loc": F0, "scale": np.abs(1e-10 * F0)},
-            "F1": {"type": "norm", "loc": F1, "scale": np.abs(1e-10 * F1)},
-            "F2": F2,
-            "Alpha": Alpha,
-            "Delta": Delta,
+            "F0": {"type": "norm", "loc": self.F0, "scale": np.abs(1e-10 * self.F0)},
+            "F1": {"type": "norm", "loc": self.F1, "scale": np.abs(1e-10 * self.F1)},
```

Comment:
I thought you said uniform? Or do you do this in order to test the normalization warning as well?

Reply:
I changed Alpha,Delta to uniform in the "skyprior" case below, but left F0,F1 alone in both.